iroh-bytes/src/protocol/range_spec.rs

```diff
@@ -41,25 +56,25 @@ impl RangeSpec {
         Self(res)
     }
 
-    /// An empty range spec
+    /// A [`RangeSpec`] deselecting the entire blob.
```

Comment:
selecting nothing from the blob?

It is called EMPTY because RangeSpec's are really just compact sets of integers, so this is the empty set.

Reply:
fixed and added